package-lock.json

```diff
@@ -1,39 +1,9581 @@
 {
   "name": "nestjs-otel",
   "version": "2.2.0",
```

Comment:
Which node version are you using? this might break stuff for node 14? Right now I thought about only supporting LTS because I could not use the latest in prod yet

Reply:
```
npm --version 
7.20.0

node --version
v14.17.2
```